db/db_impl_write.cc

```diff
@@ -1064,6 +1065,8 @@ Status DBImpl::SwitchWAL(WriteContext* write_context) {
                  oldest_alive_log, total_log_size_.load(), GetMaxTotalWalSize());
   // no need to refcount because drop is happening in write thread, so can't
   // happen while we're in the write thread
+
```

Comment:
Not sure you need to call it here.

Reply:
I will add a different callback here so that you can override it with empty body if you do not need it.